ui/src/app/pages/data/cohort/custom-funnel.tsx

```diff
@@ -215,17 +222,28 @@ export const CustomFunnel = withCurrentWorkspace()(
                     <Spinner size={30} style={{ marginTop: '0.25rem' }} />
                   ) : (
                     <>
-                      {percentage < 10 && (
+                      {(percentage < 1 && (
                         <div
                           style={{
                             position: 'absolute',
-                            left: `calc(${47 - percentage}% - 0.5rem)`,
+                            left: `calc(${42 - percentage}% - 0.5rem)`,
                             top: '0.5rem',
                           }}
                         >
-                          {funnelGroup.count.toLocaleString()}
+                          {displayText}
                         </div>
-                      )}
+                      )) ||
+                        (percentage < 10 && (
+                          <div
+                            style={{
+                              position: 'absolute',
+                              left: `calc(${47 - percentage}% - 0.5rem)`,
+                              top: '0.5rem',
+                            }}
+                          >
+                            {displayText}
+                          </div>
+                        ))}
```

Comment:
I don't think we need separate elements here for `percentage < 1` and `percentage < 10`. The `calc(${42 - percentage}% - 0.5rem)` position seems to work for both conditions. I get overlap for percentage of 1-10 how you currently have it:
![Screenshot 2023-11-15 at 9 13 25 AM](https://github.com/all-of-us/workbench/assets/40036095/a94c8061-812b-43cb-8ae9-c3ef3976955f)